### PR TITLE
[operator] new value for k8s cluster domain

### PR DESCRIPTION
**Pull request by jouve**
override k8s default cluster domain (cluster.local)

### CODE REVIEW

**Comment by Allex1** [comment]
@jouve thanks for your contribution .
Can you please bump the chart version and run `make generate-examples CHARTS=opentelemetry-operator` so that the CI can pass ?

**Comment by jouve** [comment]
done :)

**Review by Allex1 — APPROVED** [review]

File: charts/opentelemetry-operator/values.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ nameOverride: ""
 ## Reference one or more secrets to be used when pulling images from authenticated repositories.
 imagePullSecrets: []
 
+## Kubernetes cluster domain suffix
+clusterDomain: cluster.local
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/values.yaml:15]: I think we should nest this under `certManager` unless such a setting would be useful other places in the chart.

Comment (jouve) [reply charts/opentelemetry-operator/values.yaml:15]: I've found lots of charts have this as clusterDomain or global.clusterDomain, so it's less surprising there

**Comment by jouve** [comment]
how can I update the values.schema.yaml ?

**Comment by Allex1** [comment]
> how can I update the values.schema.json ?

it's mostly a manual process currently

**Review by TylerHelmuth — APPROVED** [review]
Thanks for sticking with this!

**Pull request merged**